numpy/testing/tests/test_utils.py

```diff
@@ -144,7 +145,10 @@ def test_recarrays(self):
         c['floupipi'] = a['floupi'].copy()
         c['floupa'] = a['floupa'].copy()
 
-        self._test_not_equal(c, b)
+        with suppress_warnings() as sup:
```

Comment:
So this might cause some problems downstream? I'm not terribly concerned, the only way to find out is to try.